alexaapi/request_intent.go

```diff
@@ -63,8 +63,23 @@ type SlotResolutions struct {
 	ResolutionsPerAuthority []SlotAuthority `json:"resolutionsPerAuthority,omitempty"`
 }
 
+type StatusCode string
```

Comment:
Let's make this more specific to avoid future collisions.
e.g. StatusCode -> SlotAuthorityStatusCode